src/content/structured/accessibility/needs/auditory.mdx

```diff
@@ -55,7 +55,7 @@ Don't rely on automated subtitling and transcription because it might be inaccur
 ## References
 
 1. <a
-     href=" https://www.actiononhearingloss.org.uk/about-us/our-research-and-evidence/facts-and-figures/"
+     href="https://www.actiononhearingloss.org.uk/about-us/our-research-and-evidence/facts-and-figures/"
```

Comment:
seems to redirect to https://rnid.org.uk/about-us/research-and-policy/facts-and-figures/
should that url be used instead?